modules/system/config-wfrp4e.js

```diff
@@ -1726,6 +1728,18 @@ WFRP4E.PrepareSystemItems = function() {
                             damage = scriptArgs.damage;
                             msg += await actor.applyBasicDamage(damage, {damageType : game.wfrp4e.config.DAMAGE_TYPE.IGNORE_ALL, suppressMsg : true})
 
+                            let test = await actor.setupSkill(game.i18n.localize("NAME.Endurance"), {appendTitle : " - Poison"})
+                            await test.roll();
+                            if (test.result.outcome == "success")
+                            {
+                                await actor.removeCondition("poisoned", Math.min(test.result.SL, conditionValue));
+                                msg += "<br/>Number of removed Poisoned Conditions: " + Math.min(test.result.SL, conditionValue);
+                            }
+                            else
+                            {
+                                msg += "<br/>Failed to remove Poisoned Condition";
+                            }
```

Comment:
`appendTitle` should probably use `game.i18n.localize("WFRP4E.ConditionName.Poisoned")`

You can also use `test.succeeded` and `test.failed` getters


Also, I'm not sure about this in general because I think the handling of condition removal deserves more attention, at the very least it should be decoupled from applying the damage

Reply:
ok, i thought that it might follow the same pattern as other scripts, i.e. bleeding, but maybe i will come up with something else, to remember about the tests to remove conditions.

Reply:
I think a separate script (manual trigger) for removal might work, I simply don't want to bind the effect and the removal process together.

Reply:
so, i added separaton for removal and applying damage. I like to have this reminder at end of the round, because during my games i often forget to check for players/enemies to remove conditions